README.md

```diff
@@ -144,81 +144,146 @@ The old repo is no longer the recommended approach for running and extending the
 
 Follow the instructions below to run the AI Mech demo executing the tool in `./packages/valory/customs/openai_request.py`. Note that AI Mechs can be configured to work in two modes: *polling mode*, which periodically reads the chain, and *websocket mode*, which receives event updates from the chain. The default mode used by the demo is *polling*.
 
-First, you need to configure the worker service. You need to create a `.1env` file which contains the service configuration parameters. We provide a prefilled template (`.example.env`). You will need to provide or create an [OpenAI API key](https://platform.openai.com/account/api-keys).
+> **Warning**<br />
+> ** You won't be able to run the Mech with the configurations displayed below (in particular since you will not have access to the private key for the agent address displayed), and will need to adjust them.
 
-```bash
-# Copy the prefilled template
-cp .example.env .1env
+Now, you have two options to run the worker: as a standalone agent or as a service.
 
-# Edit ".1env" and replace "dummy_api_key" with your OpenAI API key.
+### Option 1: Run the Mech as a service
 
-# Source the env file
-source .1env
+#### Configuration of the service
+
+First, you need to configure the service. 
+
+Ensure you have a file with the agent address and private key (`keys.json`). You can generate a new private key file using the Open Autonomy CLI:
+
+    ```bash
+    autonomy generate-key ethereum -n 1
+    ```
+
+This is only to create a file with the right format. You need to replace the address by your own agent address and the private key by the one of this address. You do not need to change the ledger.
+
+You need to create a `.1env` file which contains the service configuration parameters. We provide a prefilled template (`.example_agent.env`). You will need to use an [OpenAI API key](https://platform.openai.com/account/api-keys) in the configuration.
+
```

Comment:
`.example_agent.env` does not exist. Moreover, this step describes how to run as a service, not as an agent.